hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/NonblockingConsumersSupervisor.java

```diff
@@ -69,8 +68,10 @@ public NonblockingConsumersSupervisor(ConfigFactory configFactory,
                 configFactory.getIntProperty(Configs.CONSUMER_COMMIT_OFFSET_PERIOD),
                 metrics
         );
-        monitor.register(SUBSCRIPTIONS, backgroundProcess::listRunningSubscriptions);
-        monitor.register(SUBSCRIPTIONS_COUNT, backgroundProcess::countRunningSubscriptions);
+        monitor.register(SUBSCRIPTIONS, backgroundProcess::runningSubscriptionsStatus);
+        monitor.register(SUBSCRIPTIONS_COUNT, backgroundProcess::countRunningProcesses);
+
```

Comment:
Do we want to measure dying processes ? These maybe for instance processes for subscriptions that not died after some constnt time despite stop/kill signal

Reply:
I think that you added such a gauge to `ConsumerProcessSupervisor`.